File: app/src/main/java/moe/tlaster/openween/common/controls/ExClickableSpan.java
package moe.tlaster.openween.common.controls;

import android.graphics.Color;
import android.text.TextPaint;
import android.text.style.ClickableSpan;
import android.view.View;

/**
 * Created by Asahi on 2016/9/23.
 */

public class ExClickableSpan extends ClickableSpan {
    @Override
    public void onClick(View widget) {

    }

    @Override
    public void updateDrawState(TextPaint ds) {
        super.updateDrawState(ds);
        ds.setColor(Color.BLUE);
        ds.setUnderlineText(false);
    }
}

File: app/src/main/kotlin/moe/tlaster/openween/common/controls/ExClickableSpan.kt
package moe.tlaster.openween.common.controls

import android.graphics.Color
import android.text.TextPaint
import android.text.style.ClickableSpan
import android.view.View

/**
 * Created by Asahi on 2016/9/23.
 */

open class ExClickableSpan : ClickableSpan() {
    override fun onClick(widget: View) {

    }

    override fun updateDrawState(ds: TextPaint) {
        super.updateDrawState(ds)
        ds.color = Color.BLUE
        ds.isUnderlineText = false
    }
}
